tests/test_add_model.py

```diff
@@ -0,0 +1,423 @@
+from unittest.mock import MagicMock, Mock, patch
```

Comment:
Could you kindly rework this to make it python 2 compatible?  I know it's EOL, but still good to keep the code portable and compatible with existing projects.

Reply:
There's some examples in the current branch in Ran's repository.

Reply:
Yep, agreed that PY2 should be supported. Changed imports to look like:
```
try:
    from unittest.mock import patch
    from unittest.mock import MagicMock
    from unittest.mock import Mock
except ImportError:
    from mock import patch
    from mock import MagicMock
    from mock import Mock
```